src/main/java/AttendanceTracker/file.txt

```diff
@@ -0,0 +1,25 @@
+05/2/23
```

Comment:
where does this date come from? And will we need to change it for each text file?